docs/sources/mimir/configure/configure-shuffle-sharding/index.md

```diff
@@ -129,18 +129,22 @@ If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and
 
 #### Limitation: Decreasing the tenant shard size
 
-The current shuffle sharding implementation in Grafana Mimir has a limitation that prevents you from safely decreasing the tenant shard size when you enable ingesters’ shuffle sharding on the read path.
+The current shuffle sharding implementation in Grafana Mimir has a limitation that prevents you from abruptly decreasing the tenant shard size when you enable ingesters’ shuffle sharding on the read path. Decreasing the tenant shard size without following the complete procedure might result in query failures.
 
-If a tenant’s shard decreases in size, there is currently no way for the queriers and rulers to know how large the tenant shard was previously, and as a result, they potentially miss an ingester with data for that tenant.
-The blocks-storage.tsdb.retention-period, which is used to select the ingesters that might have received series since 'now - blocks-storage.tsdb.retention-period', doesn't work correctly for finding tenant shards if the tenant shard size is decreased.
+When you decrease a tenant's shard size, the queriers and rulers can't determine how large the tenant shard was previously. As a result, they could potentially miss an ingester with data for a given tenant.
```

Comment:
I think we are missing a very brief explanation of *why* this occurs, which is the whole reason we need this process in the first place.

When the `-querier.shuffle-sharding-ingesters-enabled=false`, the queriers check all ingesters for the blocks, when it is true, they check only those that are supposed to be assigned to that tenant via the shuffle-shard.

May not be worth including, but when it is off, you may get slightly increased latency and CPU usage as the queriers have to cycle through asking more ingesters for the data.